charts/graph-node/dashboards/query-performance-overview.json

```diff
@@ -1474,8 +1523,8 @@
     ]
   },
   "timezone": "",
-  "title": "Graph Query Performance Overview",
-  "uid": "graph-query-performance-overview",
+  "title": "Graph Query Performances",
+  "uid": "graph-query-performances",
```

Comment:
Would you mind renaming this to "Performance", singular?
Nice dashboard!

Reply:
ah yes too many edits! I've reverted to the initial name which is better